src/main/java/org/reaktivity/nukleus/http/internal/stream/ServerAcceptStream.java

```diff
@@ -421,6 +433,24 @@ private void processEnd(
         doEnd();
     }
 
+    private void processAbort(
+            DirectBuffer buffer,
+            int index,
+            int length)
+    {
+        factory.correlations.remove(acceptCorrelationId);
+        if (targetBeginIssued)
+        {
+            factory.writer.doAbort(target, targetId);
+            cleanupConnectReply.run();
+        }
+        else
+        {
+            factory.writer.doAbort(acceptReply, replyStreamId);
+        }
+        releaseSlotIfNecessary();
```

Comment:
So the 3 reactions to `ABORT` are independent, not nested.
1. `ABORT` accept reply
2. `ABORT` connect request if `targetBeginIssued`
3. `RESET` connect reply (by calling cleanupConnectReply) if removed correlation is non-null

Reply:
> RESET connect reply (by calling cleanupConnectReply) if removed correlation is non-null

I thought it should be `if removed correlation is NULL`

Reply:
sorry, that was a typo, should be "if removed correlation is `null`"